service-front/module/Application/view/application/pages/cp/choose_country_Id.twig

```diff
@@ -0,0 +1,50 @@
+{% extends "layout/layout" %}
+{% block content %}
+    {% include 'layout/id_check_banner.twig' with details_data %}
+    <div class="govuk-width-container govuk-!-padding-top-5">
+        <form method="POST">
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset" role="group" aria-describedby="passport-issued-hint">
```

Comment:
This aria-describedby attribute refers to a element that doesn't exist.

You should consider adding some [Twig macros](https://twig.symfony.com/doc/3.x/tags/macro.html) for common blocks of HTML (e.g. fieldsets) so you don't have to hand-crank this sort of thing every time.